pages/wishlist.html

```diff
@@ -0,0 +1,384 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="../assets/logo/logo.svg" type="image/svg+xml">
+    <link rel="stylesheet" href="https://dev-ui.netlify.app/css/devui.css">
+    <link rel="stylesheet" href="../css/reset.css">
+    <link rel="stylesheet" href="../css/navbar.css">
+    <link rel="stylesheet" href="../css/products.css">
+    <link rel="stylesheet" href="../css/wishlist.css">
+    <script src="../scripts/navbar.js" defer></script>
+    <script src="../scripts/aside_navbar.js" defer></script>
+    <title>3 on 3 Store | Get what makes you 3/3</title>
+</head>
+
+<body>
+
+    <header class="header">
+
+        <nav class="navbar">
+
+            <div class="navbar-brand">
+                <a href="/" class="nav-link">
+                    <img src="../assets/logo/logo.svg" alt="" width="40" height="40"
```

Comment:
add more descriptive `alt`

Reply:
I have added description in `alt` attribute, thanks for pointing out.